File: desktop/src/com/minimal/planet/desktop/DesktopLauncher.java
package com.minimal.planet.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.minimal.arkanoid.wrap.ArkanoidDebugGame;

import static com.minimal.planet.desktop.TexturePackKt.texturePacker;

public class DesktopLauncher {
	public static void main (String[] arg) {
		texturePacker(arg);

		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
//		new LwjglApplication(new Planet(), config);
//		new LwjglApplication(new ArkanoidGame(), config);
		new LwjglApplication(new ArkanoidDebugGame(), config);
	}
}

File: desktop/src/com/minimal/planet/desktop/DesktopLauncher.kt
package com.minimal.planet.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.minimal.arkanoid.wrap.ArkanoidDebugGame

fun main(args: Array<String>) {
    val config = LwjglApplicationConfiguration()
    config.width = 400
    config.height = 640
    LwjglApplication(ArkanoidDebugGame(), config)
}